src/components/Forms/CreateNewServiceForm.jsx

```diff
@@ -0,0 +1,146 @@
+import {ErrorMessage} from '@hookform/error-message';
+import {nanoid} from 'nanoid';
+import {useRouter} from 'next/router';
+import {useForm} from 'react-hook-form';
+
+import useServiceStore from '../../hooks/useServiceStore';
+
+import StyledForm from './StyledComponents/styledForm';
+import StyledWrapper from './StyledComponents/styledFormWrapper';
+import StyledInput from './StyledComponents/styledInput';
+import StyledLabel from './StyledComponents/styledLabel';
+import StyledSubmitButton from './StyledComponents/styledSubmitButton';
+
+export default function CreateNewServiceForm() {
+	const addService = useServiceStore(state => state.addService);
+	const router = useRouter();
+	const {
+		register,
+		formState: {errors},
+		handleSubmit,
+	} = useForm({
+		criteriaMode: 'all',
+	});
+	const onSubmit = data => {
+		let service = {
```

Comment:
Why not use a `const` here? :)
```suggestion
		const service = {
```